model/src/payout_curve/quanto.rs

```diff
@@ -369,8 +407,12 @@ pub fn bankruptcy_price_short(initial_price: u64, leverage: Leverage) -> u64 {
 ///
 /// This is an absolute value. How to apply it in order to calculate the bankruptcy price will
 /// depend on the party's position.
-fn bankruptcy_price_shift(initial_price: u64, leverage: Leverage) -> u64 {
-    initial_price / leverage.0 as u64
+/// Note: the resulting bankruptcy price is rounded to 0 decimal points.
```

Comment:
🔧 I think this line is unnecessary because the return type doesn't support decimal points.